server/src/schema.rs

```diff
@@ -33,6 +33,38 @@ table! {
     }
 }
 
+table! {
+    comment_aggregates_fast (fast_id) {
+        id -> Nullable<Int4>,
+        creator_id -> Nullable<Int4>,
+        post_id -> Nullable<Int4>,
+        parent_id -> Nullable<Int4>,
+        content -> Nullable<Text>,
+        removed -> Nullable<Bool>,
+        read -> Nullable<Bool>,
+        published -> Nullable<Timestamp>,
+        updated -> Nullable<Timestamp>,
+        deleted -> Nullable<Bool>,
+        ap_id -> Nullable<Varchar>,
+        local -> Nullable<Bool>,
+        community_id -> Nullable<Int4>,
+        community_actor_id -> Nullable<Varchar>,
+        community_local -> Nullable<Bool>,
+        community_name -> Nullable<Varchar>,
+        banned -> Nullable<Bool>,
+        banned_from_community -> Nullable<Bool>,
+        creator_actor_id -> Nullable<Varchar>,
+        creator_local -> Nullable<Bool>,
+        creator_name -> Nullable<Varchar>,
+        creator_avatar -> Nullable<Text>,
```

Comment:
re: my comments above about otherwise normalized data - just eyeballing this table def it doesn't seem like there is much value to the cols above this comment vs. the ones below.  All the user/creator/community/comment text fields or bool values can be retrieved via normal joins using the `id` values we populate here.  We can avoid _a lot_ of unneeded DB processing by just counting and not refreshing + storing all of it

Reply:
That's fair, maybe sometime I'll have time to split out the counts into their own view.